detect/synthetic-monitoring/playwright-checks/custom-dependencies.mdx

```diff
@@ -131,6 +136,28 @@ export default defineConfig({
 })
 ```
 
+## Dependency Caching
+
+Checkly caches installed dependencies between check runs to speed up execution and reduce installation time. This means your dependencies are installed once and reused for subsequent runs until your `package.json` and lock files change.
```

Comment:
I think these should be around the `-lock` files instead of the `package.json` otherwise could be confusing

Reply:
done!